src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  function add(n) {
+    calculator.result += n;
+  }
+
+  function subtract(n) {
+    calculator.result -= n;
+  }
+
+  function multiply(n) {
+    calculator.result *= n;
+  }
+
+  function divide(n) {
+    calculator.result /= n;
+  }
```

Comment:
[CODE KNOWLEDGE] It's recommended to use the method shorthand syntax when defining methods inside an object.

BAD EXAMPLE:
 divide: function(n) {
   this.result /= n;
 },

GOOD EXAMPLE:
 divide(n) {
   this.result /= n;
 },